helm/efk-stack-app/values.yaml

```diff
@@ -19,7 +19,8 @@ elasticsearch-curator:
   command: ["/usr/bin/curator"]
   image:
     repository: quay.io/giantswarm/bobrik-curator
-    tag: 5.8.1
+    tag: 5.8.1@sha256:3aaf4d8aca9ec1d77fc46e99ba119abf4ad4d5ac42eb7e1227c420a8b6927259
```

Comment:
Why do we need the SHA in the tag?

Reply:
It was suggested by @webwurst 
I thinks it's best practice security wise.

Reply:
You can push a different image under an "old" tag. This is not advised and if everyone keeps that in mind there shouldn't be a problem. But could still happen by mistake or maybe even as a security attack. The SHA is based on the content of the image manifest which keeps a list of all content hashes of the referenced layers. So when calculating the layer hashes on your local disk and end up with the same as for the image tag you can be sure to run the container as intended, no manipulation on purpose or by mistake happened in between. I don't think the container runtime really checks this before every start of a container, but at least while pulling images. Some people advise to always pull images, to avoid starting a container based on a locally manipulated image. Explicitly calculating SHA hashes should be enough if the runtime would provide that option, I guess. Some background, history and weirdness is for example here explained: [Explaining Docker Image IDs](https://windsock.io/explaining-docker-image-ids/). /cc @stone-z